### PR TITLE
Spellbook corey

### PR DESCRIPTION
**Pull request by bobcorey**

### CODE REVIEW

**Comment by bobcorey** [comment]
Added scripts for PIP Installation

File: requirements/release.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cached_property
```
Comment (lucpeterson) [inline requirements/release.txt:1]: change these to be only the libraries needed by the spell book

these are left-over from merlin

it's ok if it's empty for now

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+###############################################################################
+
+import os
+
+from setuptools import find_packages, setup
+
+version = "1.0"
```
Comment (lucpeterson) [inline setup.py:35]: Change to 0.0.1

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
+###############################################################################
+# Copyright (c) 2020, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory
+# Written by the Merlin dev team, listed in the CONTRIBUTORS file.
+# <merlin@llnl.gov>
+#
+# LLNL-CODE-797170
+# All rights reserved.
+# This file is part of Merlin, Version: 1.7.5.
```
Comment (lucpeterson) [inline setup.py:9]: update this to be Merlin-Spellbook, Version: 0.0.1

**Review by lucpeterson — CHANGES_REQUESTED** [review]
Please update the setup.py and various requirements.txt

Why are there lots of other file motions/deletions?

**Review by ben-bay — CHANGES_REQUESTED** [review]
Let's get rid of the merlin dependencies; merlin and most of its dependencies are not dependencies of merlin-spellbook.

Have you successfully tested the `setup.py` with PYPI? You may consider using something like [this](https://packaging.python.org/guides/using-testpypi/) to ensure everything is working the way you intend.

**Pull request merged**